mathics/core/expression.py

```diff
@@ -1895,7 +1948,7 @@ def __init__(self, head):
 
     def __call__(self, elements):
         expr = Expression(self._head)
-        expr.elements = tuple(elements)
+        expr.elements = elements
```

Comment:
same as before...

Reply:
Yes, in the future this should be one Expression call. I think I mentioned that somewhere else. 

It is however not something I want to undertake right now.